packages/core/helper-plugin/src/utils/awaitToJs.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Async await wrapper for easy error handling
+ *
+ * @deprecated
+ * @param {Promise<unknown>} promise - Promise or Async Function.
+ * @param {object} errorExt - Custom error message extended onto the error from the given promise.
+ * @return {Promise<unknown[] | any[]>}
```

Comment:
I don't think we need these since we have the TypeScript typings. Same things for the other files.

Reply:
I've removed the JSDoc tags except `@deprecated` from all files.